evalml/pipelines/components/transformers/encoders/onehot_encoder.py

```diff
@@ -110,6 +110,7 @@ def fit(self, X, y=None):
         top_n = self.parameters["top_n"]
         X = infer_feature_types(X)
         if self.features_to_encode is None:
+            # --> should update to not include ordinals once the ord encoder is integragted? Maybe that's configurable based on whether ordinal encoder is used?
```

Comment:
A note to myself about integrating the new component. Do we need a separate issue for actually using the component? Or do we use the same issue?

Reply:
We usually file another ticket but it's flexible!

Reply:
Created a follow-up issue: https://github.com/alteryx/evalml/issues/3744